docs/docs/markers.mdx

```diff
@@ -0,0 +1,289 @@
+---
+id: markers
+sidebar_label: Markers
+title: Markers
+description: Find out how to mark points of interest in dialogues using Marker conditions.
+abstract: Markers are conditions that allow you to describe and mark points of interest in dialogues.
+---
+
+
+
+:::caution
+
+This feature is currently experimental and might change or be removed in the future. Share your feedback in the forum to help us make it production-ready.
+
+:::
+
+## Overview
+
+Markers are conditions that allow you to describe and mark points of interest in dialogues. In Rasa, a dialogue is represented as a sequence of events,
+which include bot actions that were executed, intents that were detected, and slots that were set.
+Markers allow you to describe conditions over such events, and when these conditions are met, the dialogues are "marked" for further analysis or inspection.
+
+There are several applications for Markers. For example, they can be used to define your bot's **Key Performance Indicators (KPIs)**,
```

Comment:
Is uppercase Markers a typo?

Reply:
I use upper case when I'm referring to the feature, and lowercase when I'm referring to individual markers. Does this make sense or should I lowercase all of them?

Reply:
Right, I have no strong opinions either way 🙂